### ISSUE TITLE
get url path truncated for some reason

### ISSUE THREAD

**Issue by tansaku** [wopian/kitsu#553]
I have Kitsu working consuming a JSON API for a few different url paths, e.g. 

```js
api.get('venues/venues', {})
```
but I'm now working with a slightly longer url path:

```js
api.get('/profile/accounts/me', {})
```

but for some reason when I dump the request the `/me` isn't there.  Here's the underlying getter setup:

```js
  api (s) {
    const api = new Kitsu({
      baseURL: `${s.env.apiUrl}/${s.env.apiPrefix}`,
      pluralize: false
    })

    api.interceptors.request.use(request => {
      console.log('Starting Request', stringify(request, null, 2))
      return request
    })
    return api
  },
```

and the above console log gives the following:

```
[dev:nuxt] Starting Request {
[dev:nuxt]   "url": "/profile/accounts",
[dev:nuxt]   "method": "get",
[dev:nuxt]   "params": {},
[dev:nuxt]   "headers": {
[dev:nuxt]     "common": {
[dev:nuxt]       "Accept": "application/json, text/plain, */*"
```

when I would have thought it should give me this:

```
[dev:nuxt]   "url": "/profile/accounts/me",
```

we do have a couple of pieces of middleware in the nuxt app, but I've reviewed and can't see anything that would truncate the last part of the URL path like this.  Is there any Kitsu setting that might have this sort of effect?

I've manipulated this a few times doing things like:

```js
api.get('/fun/blah/him', {})
```
and each time the last part of the path is chopped off - in this case `/him`

**Comment by wopian** [comment]
What version of `kitsu` are you using? This should be fixed in [`9.1.19`](https://github.com/wopian/kitsu/releases/tag/v9.1.19) and [`10.0.0-alpha.11`](https://github.com/wopian/kitsu/releases/tag/v10.0.0-alpha.11).

**Comment by tansaku** [comment]
aha, sorry I should have included that - we're on 9.1.15 - I've bumped to 9.1.19 and that's fixed it - thanks so much for your help :-)
